### PR TITLE
[DotNetCore] Add Templates for VB.Net Projects

### PR DESCRIPTION
**Pull request by rodrmoya**
Taken from PR #6220 so that it gets built
> [Fixes  VSTS #698917](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/698917)

### CODE REVIEW

File: main/src/addins/MonoDevelop.DotNetCore/Properties/MonoDevelop.DotNetCore.addin.xml [hunk 1]
```diff
@@ -622,6 +762,18 @@
 			wizard="MonoDevelop.DotNetCore.ProjectTemplateWizard"
 			condition="UseNetCore22=true"
 			category="netcore/test/general" />
+		<Template
+			id="Microsoft.Test.MSTest.VisualBasic"
+			templateId="Microsoft.Test.MSTest.VisualBasic.2.2"
+			_overrideName="MSTest Project"
+			_overrideDescription="Creates a new MSTest project."
+			_overrideLanguage="VBNet"
```
Comment (mrward) [inline main/src/addins/MonoDevelop.DotNetCore/Properties/MonoDevelop.DotNetCore.addin.xml]: The language does not need to be translated. So it does not need to use an underscore at the start of its name.

**Comment by Matthew-Collins** [comment]
I noticed a couple of issues:
1) When “VBNet” is selected, in the New Project dialog, the combo box no longer fits the ui panel ?
2) MSTest and xUnit Test projects seem unstable and crash monodevelop ? Please try..

**Comment by rodrmoya** [comment]
doh, it's not, it should be an `||` not a `&&`

**Comment by rodrmoya** [comment]
@Matthew-Collins what do you mean by #1? I see the combo box growing to fit "VBNet"

**Comment by mrward** [comment]
@rodrmoya The problem with the button is something that is known. Not sure if it is filed anywhere. It used to work in older versions. There were some accessibility changes which broke it from what I remember. If you do not have VB.NET selected and then change the selection the wording is truncated.

newprojectdialog

**Comment by rodrmoya** [comment]
oh ok, don't see that UI problem though

**Comment by rodrmoya** [comment]
@Matthew-Collins as for #2, ugh, you're right, getting this when building the xUnit VB project created from the template, 1000s of these:

```  at MonoDevelop.UnitTesting.VsTest.VsTestNamespaceTestGroup.AddTest (MonoDevelop.UnitTesting.VsTest.VsTestUnitTest) [0x0009a] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/addins/MonoDevelop.UnitTesting/MonoDevelop.UnitTesting.VsTest/VsTestNamespaceTestGroup.cs:75
  at MonoDevelop.UnitTesting.VsTest.VsTestNamespaceTestGroup.AddTest (MonoDevelop.UnitTesting.VsTest.VsTestUnitTest) [0x0009a] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/addins/MonoDevelop.UnitTesting/MonoDevelop.UnitTesting.VsTest/VsTestNamespaceTestGroup.cs:75
  at MonoDevelop.UnitTesting.VsTest.VsTestNamespaceTestGroup.AddTest (MonoDevelop.UnitTesting.VsTest.VsTestUnitTest) [0x0009a] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/addins/MonoDevelop.UnitTesting/MonoDevelop.UnitTesting.VsTest/VsTestNamespaceTestGroup.cs:75
  at MonoDevelop.UnitTesting.VsTest.VsTestNamespaceTestGroup.AddTest (MonoDevelop.UnitTesting.VsTest.VsTestUnitTest) [0x0009a] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/addins/MonoDevelop.UnitTesting/MonoDevelop.UnitTesting.VsTest/VsTestNamespaceTestGroup.cs:75
  at MonoDevelop.UnitTesting.VsTest.VsTestNamespaceTestGroup.AddTest (MonoDevelop.UnitTesting.VsTest.VsTestUnitTest) [0x0009a] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/addins/MonoDevelop.UnitTesting/MonoDevelop.UnitTesting.VsTest/VsTestNamespaceTestGroup.cs:75
  at MonoDevelop.UnitTesting.VsTest.VsTestNamespaceTestGroup.AddTest (MonoDevelop.UnitTesting.VsTest.VsTestUnitTest) [0x0009a] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/addins/MonoDevelop.UnitTesting/MonoDevelop.UnitTesting.VsTest/VsTestNamespaceTestGroup.cs:75
  at MonoDevelop.UnitTesting.VsTest.VsTestNamespaceTestGroup.AddTest (MonoDevelop.UnitTesting.VsTest.VsTestUnitTest) [0x0009a] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/addins/MonoDevelop.UnitTesting/MonoDevelop.UnitTesting.VsTest/VsTestNamespaceTestGroup.cs:75
```

**Comment by rodrmoya** [comment]
NUnit ones work though, I'll have a look at the issue, if it's an easy fix, we can include it here, if not, I guess we'll have to disable those 2 templates

**Comment by mrward** [comment]
@rodrmoya I think that stackoverflow has been filed before - did not have a way to repro it before - https://github.com/mono/monodevelop/issues/5817

**Comment by slluis** [comment]
@rodrmoya The stack overflow should be fixed by https://github.com/mono/monodevelop/pull/6288. Can you please verify?

**Comment by rodrmoya** [comment]
Yes, just rebased this branch with latest master, and everything works as expected.

So, this is ready for merging now. @Matthew-Collins anything else missing?

**Comment by Matthew-Collins** [comment]
Yes all working as expected, thank you all....!

**Comment by Matthew-Collins** [comment]
@mrward support seems limited in the test runner for parameter tests in MSTest <DataTestMethod> <DataRow(True)> etc... compared to NUnit and XUnit

**Comment by mrward** [comment]
@Matthew-Collins Probably worth opening a new issue for the MSTest problem. There is no explicit code added for MSTest. Most of the test runner logic is NUnit specific.

**Pull request merged**